### PR TITLE
Create gd.js

### PR DESCRIPTION
**Pull request by MacMhicheil**
Create Gàidhlig (Scottish Gaelic) translation

### CODE REVIEW

**Comment by MacMhicheil** [comment]
Can anybody tell me why this has failed?

**Comment by ironwallaby** [comment]
You have not added tests! You will want to copy, for example, `/test_cases/en.json` to `/test_cases/gd.json`, translate all of the examples, and then make sure your translation module matches the expected results. Thanks!

**Comment by MacMhicheil** [comment]
Thanks for your help @ironwallaby!

I'm a bit puzzled why this is still failing. The log shows that there are 4 translations failing the test because of upper case letters mid word (highlighted in bold below) but these letters are all lower case in both my module and test files. They all seem to follow accented characters, but other translations with accented characters have all passed the test.

280      AssertionError: 'Ceò**T**hach' === 'Ceòthach'
281      + expected - actual
282
283      -Ceò**T**hach
284      +Ceòthach

293      AssertionError: 'Sneachda Trom (1–3 ò**I**r.)' === 'Sneachda Trom (1–3 òir.)'
294      + expected - actual
295
296      -Sneachda Trom (1–3 ò**I**r.)
297      +Sneachda Trom (1–3 òir.)

306      AssertionError: 'Tha Cothrom Ann Gum Bi Stoirmean Tà**I**rneanaich' === 'Tha Cothrom Ann Gum Bi Stoirmean Tàirneanaich'
307      + expected - actual
308
309      -Tha Cothrom Ann Gum Bi Stoirmean Tà**I**rneanaich
310      +Tha Cothrom Ann Gum Bi Stoirmean Tàirneanaich

319      AssertionError: 'Uisge Trom Agus Stoirmean Tà**I**rneanaich' === 'Uisge Trom Agus Stoirmean Tàirneanaich'
320      + expected - actual
321
322      -Uisge Trom Agus Stoirmean Tà**I**rneanaich
323      +Uisge Trom Agus Stoirmean Tàirneanaich

**Comment by MacMhicheil** [comment]
Thanks for your help @ironwallaby

File: lib/lang/gd.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  "thursday": "Diardaoin",
+  "friday": "Dihaoine",
+  "saturday": "Disathairne",
+  "next-sunday": "Didòmhnaich an ath sheachdain",
+  "next-monday": "Diluain an ath sheachdain",
+  "next-tuesday": "Dimàirt an ath sheachdain",
+  "next-wednesday": "Diciadain an ath sheachdain",
+  "next-thursday": "Diardaoin an ath sheachdain",
+  "next-friday": "Dihaoine an ath sheachdain",
+  "next-saturday": "Disathairne  an ath sheachdain",
```
Comment (ironwallaby) [inline lib/lang/gd.js]: There's a double space here... should probably be a single space?

**Comment by ironwallaby** [comment]
Sorry for the delay in getting back to you, I'm finally back in from the holidays!

This looks good, I just had one minor comment. Happy to merge it in whenever you're ready!

**Comment by MacMhicheil** [comment]
No bother. I've removed that extra space now, so happy to proceed whenever you are ready.

**Pull request merged**